process/contributing.md

```diff
@@ -89,10 +89,19 @@ A: Yes.
 
 Q: Can umbrella CLAs be established where contributors and their organizations grant rights to all ASWF projects?
 
-A: The ASWF TAC sees value in lowering the friction to contribution, and is investigating this as an option as the number of hosted projects grows. In the meantime, projects aligning around the [Linux Foundation minimal CCLA template](ccla_template.md) will ensure that the legal review for signing a CCLA is minimal.
+A: The ASWF TAC sees value in lowering the friction to contribution, and is investigating this as an option as the number of hosted projects grows. In the meantime, projects aligning around the [ASWF 2020 CCLA template] and [ASWF 2020 ICLA template] will ensure that the legal review for signing a CCLA and/or ICLA is minimal.
 
-Q: When a project uses a CLA with the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0), is it common for the [Apache CCLA](https://www.apache.org/licenses/cla-corporate.txt) to be employed?  Should ASWF projects that require a CLA and use the Apache License, Version 2.0 employ the Apache CLA or the LF minimal CLA?
+Q: When a project uses a CLA with the [Apache License, Version 2.0], is it common for [ASF Contributor Agreements] to be employed?  Should ASWF projects that require a CLA and use the [Apache License, Version 2.0] employ the Apache CLA or the LF minimal CLA?
 
-A: There is no requirement for a project to adopt an Apache CCLA with code licensed under the Apache License, Version 2.0. Many open source projects leverage the Apache License, Version 2.0 without using the Apache CCLA or sometimes without a CLA in general (in many cases use of the [DCO](https://developercertificate.org/) alone is sufficent for project communities).
+A: There is no requirement for a project to adopt [ASF Contributor Agreements] with code licensed under the [Apache License, Version 2.0]. Many open source projects leverage the [Apache License, Version 2.0] without using [ASF Contributor Agreements] or sometimes without a CLA in general (in many cases use of the [DCO](https://developercertificate.org/) alone is sufficent for project communities).
 
-The ASWF community is aiming to align around the [Linux Foundation minimal CCLA template](ccla_template.md) in cases where use of a CLA is required.
+The ASWF community is aiming to align around the [ASWF 2020 CCLA template] and [ASWF 2020 ICLA template] in cases where use of a CLA is required. Projects may also use the [Linux Foundation minimal CCLA template] and [Linux Foundation minimal ICLA template].
+
+[Apache License, Version 2.0]: http://www.apache.org/licenses/LICENSE-2.0
+[ASF Contributor Agreements]: https://www.apache.org/licenses/contributor-agreements.html
+[ASWF 2020 template]: ccla_template_aswf2020.md
+[ASWF 2020 template]: icla_template_aswf2020.md
```

Comment:
Did you mean for these to be `ASWF 2020 CCLA template` and `ASWF 2020 ICLA template`? I think the links above aren't working b/c these have the same name but don't include the CCLA / ICLA distinction.

Reply:
Good catch! Thanks :-)